src/main/java/com/wcc/platform/domain/platform/Filters.java

```diff
@@ -0,0 +1,25 @@
+package com.wcc.platform.domain.platform;
+
+import com.wcc.platform.domain.cms.attributes.EventDays;
+import com.wcc.platform.domain.cms.attributes.EventType;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+/** Filters model. */
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+@ToString
+@Getter
+public class Filters {
```

Comment:
I think `EventFilters` because we will also have for mentors, blog.. and maybe more.

Reply:
Kept it generic filters rather than EventFilters